README.md

```diff
@@ -1,28 +1,28 @@
 # <img src="/screenshots/balena-etcher.png" alt="drawing" width="60"/>Etcher-arm-32-64
-balena-etcher v1.5.109 and later compiled from [source](https://github.com/balena-io/etcher) for armhf and arm64 for the [Raspberry Pi](https://www.raspberrypi.org) and other [ARM](https://en.wikipedia.org/wiki/ARM_architecture) based [linux](https://en.wikipedia.org/wiki/Linux) computers.
->NOTE: this only works on Debian and Debian based OS's like Ubuntu.
+balena-etcher v1.5.109 and later compiled from [source](https://github.com/balena-io/etcher) for armhf and arm64 for the [Raspberry Pi](https://www.raspberrypi.org) and other [ARM](https://en.wikipedia.org/wiki/ARM_architecture) based [Linux](https://en.wikipedia.org/wiki/Linux) computers.
+>NOTE: This only works on Debian and Debian based OS's like Ubuntu.
 
 ![Etcher on rpi screenshot](/screenshots/etcher.png)
 
 
 ## Install
-It is recommended to install this version of Etcher from Pi-Apps (click the badge bellow for more info about pi-apps) <br> 
+It's recommended that you install this version of Etcher from Pi-Apps (click the badge below for more info about pi-apps) <br> 
 [![badge](https://github.com/Botspot/pi-apps/blob/master/icons/badge.png?raw=true)](https://github.com/Botspot/pi-apps)  
-but if you prefer, you can install manually: 
-1) download the .deb file for your system architecture from the [latest release](https://github.com/Itai-Nelken/Etcher-arm-32-64/releases/latest) (armhf is 32bit arm, and arm64 is 64bit arm).
-2) open the .deb file with a package installer (just double click it) if you have one installed, or open terminal in the directory where the .deb is and type 
+but if you prefer, you can install it manually: 
+1) download the .deb file for your system architecture from the [latest release](https://github.com/Itai-Nelken/Etcher-arm-32-64/releases/latest) (armhf is 32bit arm and arm64 is 64bit arm).
+2) open the .deb file with a package installer (just double click it) if you have one installed, or open terminal in the directory where the .deb file is and type 
 ```sh
 sudo apt install -y --fix-broken the-file-name.deb
 ```
-but replace `the-file-name.deb` with the name of the .deb file you downloaded (or path to it).
+but replace `the-file-name.deb` with the path and name of the .deb file you downloaded (E.G. /home/$USER/Downloads/etcher.deb).
```

Comment:
I still prefer the old spelling, but will add a example.

Reply:
when I was changing that I thought it would be changed by you :+1:

Reply:
I thought you might change that :+1: